spaceship/lib/spaceship/tunes/build_details.rb

```diff
@@ -0,0 +1,48 @@
+module Spaceship
+  module Tunes
+    # Represents the details of a build
+    class BuildDetails < TunesBase
+      # @return (String) The App identifier of this app, provided by iTunes Connect
+      # @example
+      #   "1013943394"
+      attr_accessor :apple_id
+
+      # @return (String) Link to the dSYM file (not always available)
+      #     lol, it's unencrypted http
+      attr_accessor :dsym_url
+
+      # @return [Bool]
+      attr_accessor :include_symbols
+
+      # @return [Integer]
+      attr_accessor :number_of_asset_packs
+
+      # @return [Bool]
+      attr_accessor :contains_odr
```

Comment:
Mostly a comment: I was wondering what ODR was and this is what I found:

* http://fortheloveoftech.com/2015/06/08/apple-releases-xcode-7-beta/
"The dsymutil utility can unique redundant C++ debug types based on C++’s one-definition-rule. This makes significant size reductions in .dSYM bundles for large C++ projects while preserving the same information. (11659111)"
* https://en.wikipedia.org/wiki/One_Definition_Rule